LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019, IBM (Center of Open-Source Data & AI Technologies)
```

Comment:
Separate from the current discussion, all IBM licenses should use the same copyright (ie not our team), so this (and any header) should instead be:
```
Copyright 2018 IBM Corp. All Rights Reserved.
```

Reply:
Minior question, is it Center `of` Open-Source or Center `for` Open-Source? 
https://developer.ibm.com/code/open/centers/codait/about/

Reply:
Nice catch I-Hong, though this has been removed, it's good to remember that it's Center *FOR* Open-Source